escuro_johndeniel/Countries/Countries.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Countries of the World</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <h1>Countries of the World</h1>
+    <div id="search">
+      <input type="text" id="country-input" placeholder="Enter a country" />
+      <button id="search-button">Search</button>
+    </div>
+
+    <div id="country-details"></div>
+
+    <div id="region-countries"></div>
+
+    <div id="loading-indicator" style="display: none">Loading...</div>
```

Comment:
Hi! Would you mind making new changes in naming ids? All IDs should be in snake_case, for example:

❌ id="country-input"
✅ id="country_input"

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5